api.py

```diff
@@ -54,13 +57,27 @@ def parse_args():
     return parser.parse_args()
 
 
+def initialize_db_conn():
```

Comment:
Maybe this should just go in bootstrap under initialize db?

Reply:
Yes. that's probably better.